cardano-cli/test/cardano-cli-golden/Test/Golden/CreateTestnetData.hs

```diff
@@ -103,16 +111,17 @@ golden_create_testnet_data mShelleyTemplate =
         generated'' = map (\c -> if c == '\\' then '/' else c) generated'
     void $ H.note generated''
 
-    H.diffVsGoldenFile generated'' "test/cardano-cli-golden/files/golden/conway/create-testnet-data.out"
+    bracketSem createTestnetDataOutSem $
+      H.diffVsGoldenFile generated'' "test/cardano-cli-golden/files/golden/conway/create-testnet-data.out"
```

Comment:
If this test is run once, how does concurrent execution cause it to run multiple times?

Reply:
It's executed once. The problem is that this golden file is shared between multiple test cases, which are executed concurrently. So when we're testing golden files regeneration, two test cases see the file missing so they try to recreate file at the same time. This somehow is an issue on windows, which results in an exception here.